drake/bindings/swig/rbtree.i

```diff
@@ -0,0 +1,62 @@
+%module(package="pydrake") rbtree
+
+%include "exception_helper.i"
+%include <std_string.i>
+%include <windows.i>
+#define DRAKERBM_EXPORT
+
+%{
+#ifdef SWIGPYTHON
+  #define SWIG_FILE_WITH_INIT
+  #include <Python.h>
+#endif
+#include "RigidBodyTree.h"
+%}
+
+%include <typemaps.i>
+%include <std_vector.i>
+%include <eigen.i>
+%include <autodiff.i>
+%import <autodiffutils.i>
+
+%template(vectorVectorXd) std::vector<Eigen::VectorXd>;
+%template(vectorMatrixXd) std::vector<Eigen::MatrixXd>;
+%template(vectorString) std::vector<std::string>;
```

Comment:
We have to name them something for Swig to be happy, but it will correctly handle the conversion to python lists internally even without us using the name